### PR TITLE
feat: figure out binaries upload

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says

### CODE REVIEW

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -400,6 +400,12 @@ jobs:
       labels: [self-hosted, Windows, signtool]
 
     steps:
+      - name: Checkout repository pyansys-geometry-binaries
```
Comment (PipKat) [inline .github/workflows/ci_cd.yml:403]:
```suggestion
      - name: Check out repository pyansys-geometry-binaries
```

File: doc/source/assets.rst [hunk 2]
```diff
@@ -63,11 +63,21 @@ meaning that certain operations are not available or fail.
 Windows container
 ^^^^^^^^^^^^^^^^^
 
-* `Latest Geometry service binaries for Windows containers <https://github.com/ansys/pyansys-geometry/releases/latest/download/windows-binaries.zip>`_
+.. note::
+
+   These binaries are only available for download to users with access to
+   https://github.com/ansys/pyansys-geometry-binaries
```
Comment (PipKat) [inline doc/source/assets.rst:69]:
```suggestion
   Only users with access to ``https://github.com/ansys/pyansys-geometry-binaries`` can download these binaries.
```
You might want to make the location a link.

File: doc/source/assets.rst [hunk 3]
```diff
@@ -63,11 +63,21 @@ meaning that certain operations are not available or fail.
...
 * `Latest Geometry service Dockerfile for Windows containers <https://github.com/ansys/pyansys-geometry/releases/latest/download/windows-dockerfile.zip>`_
 
 Linux container
 ^^^^^^^^^^^^^^^
 
-* `Latest Geometry service binaries for Linux containers <https://github.com/ansys/pyansys-geometry/releases/latest/download/linux-binaries.zip>`_
+.. note::
+
+   These binaries are only available for download to users with access to
+   https://github.com/ansys/pyansys-geometry-binaries
```
Comment (PipKat) [inline doc/source/assets.rst:80]: Same change here as earlier.

File: doc/source/getting_started/docker/linux_container.rst [hunk 4]
```diff
@@ -128,7 +128,13 @@ Prerequisites
 * Download the `latest Linux Dockerfile <https://github.com/ansys/pyansys-geometry/blob/main/docker/Dockerfile.linux>`_.
 
 * Download the `latest release artifacts for the Linux
-  Docker container (ZIP file) <https://github.com/ansys/pyansys-geometry/releases/latest/download/linux-binaries.zip>`_.
+  Docker container (ZIP file) according to your version <https://github.com/ansys/pyansys-geometry-binaries/>`_.
+
+.. note::
+
+   These binaries are only available for download to users with access to
+   https://github.com/ansys/pyansys-geometry-binaries
```
Comment (PipKat) [inline doc/source/getting_started/docker/linux_container.rst:136]: And same change here again.

File: doc/source/getting_started/docker/windows_container.rst [hunk 5]
```diff
@@ -115,7 +115,13 @@ Prerequisites
 * Download the `latest Windows Dockerfile <https://github.com/ansys/pyansys-geometry/blob/main/docker/Dockerfile.windows>`_.
 
 * Download the `latest release artifacts for the Windows
-  Docker container (ZIP file) <https://github.com/ansys/pyansys-geometry/releases/latest/download/windows-binaries.zip>`_.
+  Docker container (ZIP file) according to your version <https://github.com/ansys/pyansys-geometry-binaries/>`_.
```
Comment (PipKat) [inline doc/source/getting_started/docker/windows_container.rst:118]:
```suggestion
  Docker container (ZIP file) for your version <https://github.com/ansys/pyansys-geometry-binaries/>`_.
```

File: doc/source/getting_started/docker/windows_container.rst [hunk 6]
```diff
@@ -115,7 +115,13 @@ Prerequisites
...
+
+.. note::
+
+   These binaries are only available for download to users with access to
```
Comment (PipKat) [inline doc/source/getting_started/docker/windows_container.rst:122]: Same change as earlier.

**Review by PipKat — APPROVED** [review]
Only suggested change is for that note that appears multiple times. It's much easier to understand when written in the active voice.

**Comment by RobPasMue** [comment]
Thanks for the review @PipKat!

**Pull request merged**